core/app/models/spree/pricers/conservative_line_item_pricer.rb

```diff
@@ -0,0 +1,23 @@
+module Spree
+  module Pricers
+    class ConservativeLineItemPricer
```

Comment:
I think it does. The pricer I want to see will change the price even if it's already present, and I'd like to make that the `standard` one. It's not there yet...